src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -168,10 +168,15 @@
 
     private GenomeLocSortedSet genomeLocSortedSet;
 
+    // information to keep track of when validating a GVCF
+    private SimpleInterval previousInterval;
+
+    SAMSequenceDictionary dict;
+
     @Override
     public void onTraversalStart() {
         if (VALIDATE_GVCF) {
-            SAMSequenceDictionary dict = getBestAvailableSequenceDictionary();
+            dict = getBestAvailableSequenceDictionary();
```

Comment:
I wouldn't store this, I would just call getBestAvailableSequenceDictionary() again when you need it again.   It's not used in any sort of tight loop, so the expense is negligible.  Adding fields should always be a last resort.